eng/Versions.props

```diff
@@ -116,7 +116,7 @@
     <MicrosoftmacOSRefVersion>12.3.300-rc.3.83</MicrosoftmacOSRefVersion>
     <MicrosoftMetadataVisualizerVersion>1.0.0-beta3.21075.2</MicrosoftMetadataVisualizerVersion>
     <MicrosoftNETBuildExtensionsVersion>2.2.101</MicrosoftNETBuildExtensionsVersion>
-    <MicrosoftNETCorePlatformsVersion>2.1.2</MicrosoftNETCorePlatformsVersion>
+    <MicrosoftNETCorePlatformsVersion>5.0.0</MicrosoftNETCorePlatformsVersion>
```

Comment:
@dotnet/roslyn-infrastructure any issues with this?  It was required for the new ICSharpCodeDecompiler version.  It also matches the line below which makes me feel like it's likely ok :D

Reply:
LGTM! @dibarbet fyi to flag the insertion, if there are issues.